platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/rest/HttpServletProxyResponse.java

```diff
@@ -138,9 +138,12 @@ public void flush() throws IOException {
 
         @Override
         public void close() throws IOException {
-            flush();
-            if (writer != null) {
-                writer.close();
+            try {
+                flush();
+            } finally {
+                if (writer != null) {
+                    writer.close();
```

Comment:
AFAIK Writer::close throws IOException, so there is posibility that exception is thrown by flush method will be suppressed.